specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-10-01/netapp.json

```diff
@@ -0,0 +1,2710 @@
+          "description": "The service level of the file system",
+          "enum": [
+            "Standard",
+            "Premium",
+            "Ultra"
+          ],
+          "x-ms-enum": {
+            "name": "ServiceLevel",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Standard",
+                "description": "Standard service level"
+              },
+              {
+                "value": "Premium",
+                "description": "Premium service level"
+              },
+              {
+                "value": "Ultra",
+                "description": "Ultra service level"
+              }
+            ]
+          },
+          "example": "Ultra",
+          "default": "Premium"
+        },
+        "usageThreshold": {
+          "title": "usageThreshold",
+          "type": "integer",
+          "format": "int64",
+          "description": "Maximum storage quota allowed for a file system in bytes. This is a soft quota used for alerting only. Minimum size is 100 GiB. Upper limit is 100TiB. Specified in bytes.",
+          "minimum": 107374182400,
+          "maximum": 109951162777600,
+          "default": 107374182400,
+          "example": 107374182400
+        },
+        "exportPolicy": {
+          "title": "exportPolicy",
+          "description": "Set of export policy rules",
+          "properties": {
+            "rules": {
+              "title": "Export policy rule",
+              "description": "Export policy rule",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/exportPolicyRule"
+              }
+            }
+          }
+        },
+        "protocolTypes": {
+          "title": "protocolTypes",
+          "description": "Set of protocol types",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": [
+            "NFSv4.1"
+          ]
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure lifecycle management"
+        },
+        "snapshotId": {
+          "title": "Snapshot ID",
+          "type": "string",
+          "description": "UUID v4 or resource identifier used to identify the Snapshot.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}|(\\\\?([^\\/]*[\\/])*)([^\\/]+)$",
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9760acf5-4638-11e7-9bdb-020073ca3333"
+        },
+        "baremetalTenantId": {
+          "title": "Baremetal Tenant ID",
+          "type": "string",
+          "readOnly": true,
+          "description": "Unique Baremetal Tenant Identifier.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9560acf5-4e3a-12e7-
```

Comment:
Does the user have to provide this or is it pulled from the remote volume and populated by the RP in a read-only manner?  Seems redundant to make the user specify it since they are already supplying the full remote volume resource ID which contains this info

Reply:
This is a required field that must be provided by the user. It is not available elsewhere.